### ISSUE TITLE
0x9633FB33 (U+23ADB) glyph in GB 18030 should be modified to match PRC conventions

### ISSUE THREAD

**Issue by eisoch** [eisoch/irg#34]
GB 18030
![image](https://user-images.githubusercontent.com/24350864/38137920-55ca0710-345a-11e8-87b1-9c80d381f43a.png)
The left component should be 豪. 
The TA-553A glyph in CNS11643 should be modified correspondingly.

**Comment by kenlunde** [comment]
Another case of the "UCS2003" representative glyph causing ill effect? (Though the H-Source representative glyph is the same, and is an H-only ideograph.)

**Comment by hfhchan** [comment]
Are you sure the left side is 豪?
I cannot find any evidence for this character at all...

**Comment by hfhchan** [comment]
The Cantonese pronunciation suggests it is 毅.

**Comment by eisoch** [comment]
Yes, it's the variant of 毅. So many people who don't know 毅 would read 毅 as 豪, so the left component should be written as 豪.

**Comment by kenlunde** [comment]
This particular character was included in Hong Kong GCCS, which is the predecessor of Hong Kong SCS (HKSCS).

**Comment by eisoch** [comment]
I have a friend. Her name is 某毅敏. And so many people call her 某豪敏, because they treated the character as 豪.

**Comment by eisoch** [comment]
I think the designer(s) of PingFang will agree with me. 😉 The left side of this glyph in PingFang is 豪.

**Comment by kenlunde** [comment]
This discussion suggests that the H-Source representative glyph for U+23ADB 𣫛 should be ⿰豪殳, and that Hong Kong SCS inherited the error from Hong Kong GCCS.
